### PR TITLE
added more comments to demo

### PR DESCRIPTION
**Pull request by mlyashko**
Added disclaimers for the demos that these were tested with some specific models only on VPU devices.

@vmatveic , please review (and invite others for review)

### CODE REVIEW

**Comment by kchechil** [comment]
@mlyashko who has tested all these demos on HDDL/FPGA?

File: demos/security_barrier_camera_demo/README.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Options:
     -fps                       Optional. Set the playback speed not faster than the specified FPS. 0 removes the upper bound.
     -n_wt                      Optional. Set the number of threads including the main thread a Worker class will use.
     -display_resolution        Optional. Specify the maximum output window resolution.
-    -tag                       Optional. For HDDL plugin only. Running each network on a set of Intel(R) Movidius(TM) X VPUs with a specific tag. You must specify the number of VPUs for each network in the hddl_service.config file. Refer to the corresponding README file for more information.
+    -tag                       Optional. Required for HDDL plugin only. If not set, the performance on Intel(R) Movidius(TM) X VPUs will not be optimal. Running each network on a set of Intel(R) Movidius(TM) X VPUs with a specific tag. You must specify the number of VPUs for each network in the hddl_service.config file. Refer to the corresponding README file for more information.
```
Comment (Wovchena) [inline demos/security_barrier_camera_demo/README.md]: `REAME.md` must stay consistent with demo's help message. `security_barrier_camera_demo.hpp` should also be updated.

File: demos/crossroad_camera_demo/README.md [hunk 2]
```diff
@@ -85,6 +85,11 @@ If Person Attributes Recognition or Person Reidentification Retail are enabled,
 	* **Person Attributes Recognition time** - Inference time of Person Attributes Recognition averaged by the number of detected persons.
 	* **Person Reidentification time** - Inference time of Person Reidentification averaged by the number of detected persons.
 
+> **NOTE**: On VPU devices (Intel® Movidius™ Neural Compute Stick, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs) this demo has been tested on the following Model Downloader available topologies: 
+`person-vehicle-bike-detection-crossroad-0078`
```
Comment (Wovchena) [inline demos/crossroad_camera_demo/README.md]: I guess `>*` is missing.

File: demos/text_detection_demo/README.md [hunk 3]
```diff
@@ -64,9 +64,10 @@ For example, use the following command line command to run the application:
 
 The demo uses OpenCV to display the resulting frame with detections rendered as bounding boxes and text.
 
-> **NOTE**: On VPU devices (Intel® Movidius™ Neural Compute Stick, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs) this demo is not supported with any of the Model Downloader available topologies. Other models may work incorrectly on these devices as well.
 
 ## See Also
+> **NOTE**: On VPU devices (Intel® Movidius™ Neural Compute Stick, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs) this demo is not supported with any of the Model Downloader available topologies. Other models may produce unexpected result on these devices as well.
```
Comment (IRDonch) [inline demos/text_detection_demo/README.md]: I think you accidentally moved this into the wrong section. It doesn't belong in "See Also".

File: demos/crossroad_camera_demo/README.md [hunk 4]
```diff
@@ -85,6 +85,11 @@ If Person Attributes Recognition or Person Reidentification Retail are enabled,
...
+>* `person-attributes-recognition-crossroad-0230`
+>* `person-reidentification-retail-0079`
+> Other models may produce unexpected result on these devices.
```
Comment (IRDonch) [inline demos/crossroad_camera_demo/README.md]: results*

File: demos/gaze_estimation_demo/README.md [hunk 5]
```diff
@@ -85,10 +85,12 @@ The following keys are supported:
...
 > **NOTE**: On VPU devices (Intel® Movidius™ Neural Compute Stick, Intel® Neural Compute Stick 2, and Intel® Vision Accelerator Design with Intel® Movidius™ VPUs) this demo has been tested on the following Model Downloader available topologies: 
-* `face-detection-retail-0004`
-* `head-pose-estimation-adas-0001`
-* `facial-landmarks-35-adas-0002`
-All other models may work incorrectly on these devices.
+>* `gaze-estimation-adas-0002`
+>* `face-detection-retail-0004`
+>* `face-detection-adas-0001`
+>* `head-pose-estimation-adas-0001`
+>* `facial-landmarks-35-adas-0002`
```
Comment (IRDonch) [inline demos/gaze_estimation_demo/README.md]: I think it would be preferable to list these in alphabetical order.

**Comment by AndrewBakalinIntel** [comment]
I made all the fixes in the separate MR: https://github.com/opencv/open_model_zoo/pull/245

**Comment by IRDonch** [comment]
This PR was superseded, so closing.

**Pull request closed without merging**